srcpkgs/tldr/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'tldr'
+pkgname=tldr
+version=0.6.1
+revision=1
+build_style=go
+go_import_path="github.com/isacikgoz/tldr"
+hostmakedepends="git"
+short_desc="Fast and interactive TLDR client written in Go"
+maintainer="Nathan Owens <ndowens04@gmail.com>"
+license="MIT"
+homepage="https://isacikgoz.me/tldr/"
+distfiles="https://github.com/isacikgoz/tldr/archive/v${version}.tar.gz"
+checksum=e377942c6dd9cbe3f9f55b7614e17b7c5f0694b326ad3993d0cde59aca8dc8fa
+alternatives="tldr:tldr:/usr/bin/gtldr"
```

Comment:
`conflicts=tealdear<=1.2.0_1`

Reply:
Why would we need a conflicts now? Both are being alt'd

Reply:
partial update.

Reply:
to be clearer: tealdeer in older versions than the one this PR contains conflicts with this package, because it didn't rename the binary. Adding this conflict ensures that you can't install this package without updating tealdeer first

Reply:
@sgn thanks for catching this, didn't  think about it.

Reply:
Done